config.default.yaml

```diff
@@ -62,6 +62,8 @@ cluster_options:
     exclude_carriers: []
     remove_stubs: true
     remove_stubs_across_borders: true
+    e_threshold_drop_isolated: 10 # [GWh]
+    e_threshold_merge_isolated: 10 # [GWh]
```

Comment:
Improve the comment to specify  that is a demand threshold below which isolated buses are dropped and below are merged.
1 MW (10GWh) for drop is ok and I'd merge all below 10MW.

Reply:
Added a clarification and revised the values